proxysql/dockerdir/entrypoint.sh

```diff
@@ -60,6 +87,12 @@ if [ -f "$CA" ] && [ -f "$KEY" ] && [ -f "$CERT" ] && [ -n "$PXC_SERVICE" ]; the
     sed "s^ssl_p2s_ca=\"\"^ssl_p2s_ca=\"$CA\"^"             ${PROXY_CFG} 1<> ${PROXY_CFG}
     sed "s^ssl_p2s_key=\"\"^ssl_p2s_key=\"$KEY\"^"          ${PROXY_CFG} 1<> ${PROXY_CFG}
     sed "s^ssl_p2s_cert=\"\"^ssl_p2s_cert=\"$CERT\"^"       ${PROXY_CFG} 1<> ${PROXY_CFG}
+	# Percona scheduler
+	sed "s:^sslCa.*=.*\"$:sslCa = \"${CA##*/}\":" ${PERCONA_SCHEDULER_CFG} | \
+	sed "s:^sslKey.*=.*\"$:sslKey = \"${KEY##*/}\":" | \
+	sed "s:^sslClient.*=.*\"$:sslClient = \"${CERT##*/}\":" > ${temp_proxy_scheduler_cfg}
+	cp ${temp_proxy_scheduler_cfg} ${PERCONA_SCHEDULER_CFG}
```

Comment:
```suggestion
	mv ${temp_proxy_scheduler_cfg} ${PERCONA_SCHEDULER_CFG}
```

Reply:
Thanks! Fixed: fa1754abdbfffbea1c364d1cd33689288d881aea

Reply:
Actually I have reverted the `mv` because it doesn't work :) `mv` tries to remove the destination file and we don't have permissions to remove `/etc/config.toml` so it fails:
```
+ mv /tmp/tmp.5iM8zhGtfR /etc/config.toml
mv: cannot move '/tmp/tmp.5iM8zhGtfR' to '/etc/config.toml': Permission denied

bash-4.4$ mv /tmp/tmp.T75LqOVj2M /etc/config.toml
mv: cannot move '/tmp/tmp.T75LqOVj2M' to '/etc/config.toml': Permission denied

bash-4.4$ ls -alh /tmp/tmp.T75LqOVj2M
-rw-rw-r-- 1 proxysql proxysql 4.4K Nov  4 13:09 /tmp/tmp.T75LqOVj2M

bash-4.4$ ls -alh /etc/config.toml
-rw-rw-r-- 1 proxysql proxysql 4.4K Nov  4 13:09 /etc/config.toml

bash-4.4$ cp /tmp/tmp.T75LqOVj2M /etc/config.toml

bash-4.4$ ls -alh /etc/config.toml
-rw-rw-r-- 1 proxysql proxysql 4.4K Nov  4 13:18 /etc/config.toml

bash-4.4$ rm /etc/config.toml
rm: cannot remove '/etc/config.toml': Permission denied
```